eng/scripts/Language-Settings.ps1

```diff
@@ -89,6 +90,11 @@ function Publish-javascript-GithubIODocs ($DocLocation, $PublicArtifactLocation)
   foreach ($Item in $PublishedDocs) 
   {
     $PkgName = "azure-$($Item.BaseName)"
+    # Override package name from other scope. We don't have package name to parse this out here.
```

Comment:
A few lines below we call RetrieveReleaseTag which I believe actually parse the packages and if we refactor that a little we can also use it to get the package name.

Reply:
Thanks @weshaggard . I have made the change to retrieve package name from artifact. For now I have made the change in common script within JS repo itself. I will submit another PR for common tools repo that we can merge after release is completed.